tests/unit/test_search.py

```diff
@@ -0,0 +1,45 @@
+import unittest
+from spotipy.oauth2 import SpotifyClientCredentials
+import spotipy
+import traceback
+
+
+class artist_info_test_case(unittest.TestCase):
+    def test_artist_info_with_search_str(self):
+        search_str = 'Radiohead'
+        sp = spotipy.Spotify(client_credentials_manager=SpotifyClientCredentials(
+            client_id='id_client', client_secret='secret_client'))

```

Comment:
I doubt this works with incorrect client id and secret. Even if it would work, requests in unit tests should be mocked. Real network requests can be made in integration tests however.